sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/block/CompactBlockDownloader.kt

```diff
@@ -178,6 +178,13 @@ open class CompactBlockDownloader private constructor(val compactBlockRepository
         maxEntries = maxEntries
     )
 
+    /**
+     * Returns a stream of information about roots of subtrees of the Sapling and Orchard note commitment trees.
+     *
+     * @return a flow of information about roots of subtrees of the Sapling and Orchard note commitment trees.
```

Comment:
Do we expect a flow of data or just a single Response here? The implementation returns 
 `Response<TreeStateUnsafe>` so we can tweak this documentation comment a bit.